tests/e2e/utils/utils.py

```diff
@@ -140,13 +140,14 @@ def get_cfn_client(region):
     return boto3.client("cloudformation", region_name=region)
 
 
-def get_s3_client(region):
-    return boto3.client("s3", region_name=region)
+def get_s3_client(region, access_key_id=None, secret_access_key_id=None):
+    return boto3.client("s3", region_name=region, aws_access_key_id=access_key_id,aws_secret_access_key=secret_access_key_id)
```

Comment:
why is this not necessary for the other clients? also why is it not necessary to pass credentials into the s3 client in the cleanup script, while it is necessary in the create script?

Reply:
Not 100% sure on this the original script uses s3 in this way so I kept the functionality, I haven't tested the case where we dont provide these values. They may serve no purpose here and be included by the original pr author by mistake, need further testing